hoomd/md/pytest/test_potential.py

```diff
@@ -471,12 +542,19 @@ def test_attached_params(simulation_factory, lattice_snapshot_factory,
                                                      len(snap.particles.types),
                                                      snap.particles.N)
     sim = simulation_factory(snap)
+    _check_for_skip(sim, valid_params.pair_potential)
     sim.operations.integrator = hoomd.md.Integrator(dt=0.005)
     sim.operations.integrator.forces.append(pot)
     sim.run(1)
     _assert_equivalent_type_params(pot.params.to_dict(),
                                    valid_params.pair_potential_params)
 
+def _check_for_skip(sim, pair_potential):
+    """ Determines if the simulation is able to run this pair potential """
+
+    if isinstance(sim.device, hoomd.device.GPU) and sim.device.communicator.num_ranks > 1 and \
+            str(pair_potential).startswith('<class \'hoomd.md.many_body'):
```

Comment:
This is a bit of a rough hack -- we should use `isinstance(pair_potential, Triplet)` or something similar.